File: app/src/main/java/com/jcaseydev/bart/Model2/Stations/StationList.java
package com.jcaseydev.bart.Model2.Stations;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import java.util.List;

public class StationList {

  private int id;

  @SerializedName("root")
  @Expose
  private StationListRoot root;

  public int getId() {
    return id;
  }

  public StationListRoot getRoot() {
    return root;
  }

  public List<Station> getStations() {
    return root.getStations().getStation();
  }

}

File: app/src/main/java/com/jcaseydev/bart/Model2/Stations/StationList.kt
package com.jcaseydev.bart.Model2.Stations

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class StationList {

    val id: Int = 0

    @SerializedName("root")
    @Expose
    val root: StationListRoot? = null

    val stations: List<Station>
        get() = root!!.stations?.station!!

}
